### PR TITLE
Adding Datadog steps

### PR DESCRIPTION
**Pull request by jonocairns**

### CODE REVIEW

File: docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+
+Move into the directory that was just created.  In a Windows OS (and assuming you used HOL as the directory name), you can use this command:
+
+	cd HOL
+
+**Step 2.** Create a new setting for our datadog API key and base URL.
+
+![](<media/config-section.png>)
+
+**Step 3.** Create the configuration settings initializer
```
Comment (steven2000nz) [inline docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md]: Where to add those files?

File: docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+        ...
+
+        services.AddScoped<IDataDogSettings>(p => new ConfigurationDataDogSettings(Configuration.GetSection(ConfigurationPath.Combine("Keys", "DataDog"))));
+
+        ...
+    }
+
+
+**Step 5.** Awesome. Now we want to create the contract we're using to communicate with DataDog. This can be found in the DataDog API -> http://docs.datadoghq.com/api/?lang=console#events. Create a new class called DataDogEventRequest.cs
+
```
Comment (steven2000nz) [inline docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md]: Is that possible to use Datadog-client?

Comment (jonocairns) [inline docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md]: The datadog client a) is not supported in dotnetcore and b) requires an agent on the host VM to send the reports to.

File: docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+	git clone https://github.com/Microsoft/PartsUnlimited.git HOL
+
+After a few seconds of downloading, all of the code should now be on your local machine.
+
+Move into the directory that was just created.  In a Windows OS (and assuming you used HOL as the directory name), you can use this command:
+
+	cd HOL
+
+**Step 2.** Create a new setting for our datadog API key and base URL.
```
Comment (jonocairns) [inline docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md]: done

File: docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md (hunk 1 above)
Comment (jonocairns) [inline docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md]: done

File: docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+    }
+
+**Step 11.** Let's also log when a user has a failed login attempt. Navigate to the AccountController. Add the IEventLogger in to the constructor and assign it to a private variable (see below)
+
+    public class AccountController : Controller
+    {
+        private readonly IEventLogger _eventLogger;
+
+        public AccountController(UserManager<ApplicationUser> userManager, SignInManager<ApplicationUser> signInManager, IEventLogger eventLogger)
+        {
```
Comment (steven2000nz) [inline docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md]: Do we need to register IEventLogger in Container?

**Comment by steven2000nz** [comment]
Use this branch for MS PR review.

File: docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md (hunk 4 above)
Comment (jonocairns) [inline docs/HOL-Error_Reporting_and_Monitoring_with_DataDog/README.md]: Done

**Pull request closed without merging**